megalinter/reporters/UpdatedSourcesReporter.py

```diff
@@ -63,6 +64,11 @@ def produce_report(self):
                 f" in folder {updated_sources_dir}.\n"
                 "Download it from artifacts then copy-paste it in your local repo to apply linters updates"
             )
+
+            repo = git.Repo(os.path.realpath(self.github_workspace))
+            repo.git.add(update=True)
+            repo.index.commit("megalinter auto fixes")
+            repo.git.push
```

Comment:
Also add try except, I don't want the whole MegaLinter to crash in case UpdatedSourcesReporter crashes, I prefer to just have a warning message ^^

Reply:
put a try except around which will hopefully give the correct git error message as I can't imagine this is going to work first attempt 😆